crates/optimism/rpc/src/engine.rs

```diff
@@ -40,6 +40,15 @@ pub const OP_ENGINE_CAPABILITIES: &[&str] = &[
     "engine_signalSuperchainV1",
 ];
 
+/// OP Stack protocol version
+pub const OP_STACK_SUPPORT: ProtocolVersion = ProtocolVersion::V0(ProtocolVersionFormatV0 {
```

Comment:
can we reference <https://github.com/ethereum-optimism/op-geth/blob/c3a989eb882d150a936df27bcfa791838b474d55/params/superchain.go#L13-L13> here so we're able to find this